Frameworks/include/CGContextCairo.h

```diff
@@ -138,7 +138,7 @@ class CGContextCairo : public CGContextImpl {
     virtual bool CGContextIsPointInPath(bool eoFill, float x, float y);
     virtual CGPathRef CGContextCopyPath(void);
 
-    virtual void CGContextDrawGlyphRun(const DWRITE_GLYPH_RUN* glyphRun);
+    virtual void CGContextDrawGlyphRun(const DWRITE_GLYPH_RUN* glyphRun, float lineHeight);
```

Comment:
>lineHeight [](start = 79, length = 10)

My only concern is, I don't know if this is a good name. lineHeight to me implies the distance from the bottom of the line to the top of the line, which would be ascent + descent. (ascent - descent)/2 would be the distance between the baseline and the center, but there'd be a weird burden on the caller to know that.

Probably the best way is just to pass both ascent and descent and leave the calculations in CGContextDrawGlyphRun?